dino_catalog/dinoviewer.rb

```diff
@@ -0,0 +1,90 @@
+def view_home
+  puts @messages if @messages
+  @messages = ""
+  puts "Hello there Dinosaur Hunter! What would you like to do today??"
+  puts "--- Type 'view' to see your Dinosaurs."
+  puts "--- Type 'add' to add a Dinosaur."
+  puts "--- Type 'search' to pull up Dinosaur Search Functions."
+  puts "--- Type 'save' to save your Dinosaurs."
+  puts "--- Type 'exit' to exit the program."
+  input = gets.downcase.chomp
```

Comment:
Views are supposed to handle displaying info to the user. Getting input from the user is probably controller logic.